changedetectionio/tests/test_jsonpath_selector.py

```diff
@@ -138,6 +153,36 @@ def set_modified_response():
 
     return None
 
+def test_check_json_without_filter(client, live_server):
+    json_filter = ''
+
```

Comment:
mm please add a comment here like

"Request a JSON document from a application/json source containing HTML and be sure it doesnt get chewed up by instriptis"